src/server.rs

```diff
@@ -220,27 +233,37 @@ impl CoAPServer {
     }
 }
 
-fn transmit_handler(tx_recv: RxQueue, tx_only: UdpSocket) {
-    // Note! We should only transmit with this UDP Socket
+fn response_handler(tx_recv: &RxQueue, tx_only: &UdpSocket) {
     // TODO: Add better support for failure detection or logging
```

Comment:
I think we can remove this comment now, though we should probably consider how to flatten this block out. It gets nested pretty deeply here.